docs/source/tutorials.rst

```diff
@@ -1,4 +1,25 @@
 Tutorials
 ---------
+Tutorials page is a collection of instructional materials designed to help developers learn
+how to use DFF to build conversational agents. The tutorials cover a range of topics,
+from getting started with DFF to more advanced topics such as integrating external APIs.
+Each tutorial includes detailed explanations and code examples. Tutorials cover different aspects
+of the framework and organized into sections.
 
-Work in progress... Examples of tutorials will be available in this `repository <https://github.com/deeppavlov/dialog_flow_demo>`_.
\ No newline at end of file
+The Context Storages section describes how to use context stores in DFF.
+The Messengers section covers how to use the Telegram messenger with DFF.
+The Pipeline section teaches the basics of the pipeline concept, how to use pre- and postprocessors,
+asynchronous groups and services, custom messenger interfaces, and extra handlers and extensions.
+The Script section covers the basics of the script concept, including conditions, responses, transitions,
+and serialization. It also includes tutorials on pre-response and pre-transitions processing.
+Finally, the Utils section covers the cache and LRU cache utilities in DFF.
+
+The main difference between Tutorials and Examples is that Examples typically show how to implement
+a specific feature or solve a particular problem, whereas Tutorials provide a more
+comprehensive overview of how to build a complete application.
+
```

Comment:
What about link to the `repository <https://github.com/deeppavlov/dialog_flow_demo>`_.

Reply:
The link has been added to the Examples section. Would it be better to move it to Tutorials?